code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -225,6 +230,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 			to_chat(H, "<span class='warning'>\The [blocked] is in the way!</span>")
 			return 1
 		to_chat(H, "<span class='notice'>You take a drag on your [name].</span>")
+		playsound(src, 'sound/items/cigs_lighters/inhale.ogg', 50, 0, -1)

```

Comment:
inb4 being deafened by the sound of someone inhaling becomes a meme

Reply:
This sound only plays when somebody does the specific interaction action of "taking a drag" (Click self with cigarette in hand) so it's an active sound, not continuous.
Before testing I originally had it in the smoke() proc  so it was continuous while a cig was equipped and yes it did sound like a panicking asthmatic